### PR TITLE
feat/ KB additional setting en-zh wiki

### PR DESCRIPTION
**Pull request by itIsJim**

### CODE REVIEW

**Review by xuchen — COMMENTED** [review]
In "Manual Entry" you've written about this PR:

image

Is there a reason to put it under "Additional Features"?

**Comment by xuchen** [comment]
> @xuchen I put the wiki in the new section because i thought that it was related to the button setting which is under this section and it will also explain weights and other features related to KB.

but i don't understand why we can't write it in the original section? It's on the same page as the original section.

**Comment by itIsJim** [comment]
@xuchen I can put the article back. I just moved it there to see if this arrangement was appropriate.

**Comment by xuchen** [comment]
yeah pls put it back until you prove me wrong D:

**Comment by amy17519** [comment]
the screenshot here does not make sense. Please change to a better conversation example
Screenshot 2024-06-05 at 21 04 24

File: content/en/seachat/seachat-manual/08-add-webpage-link-in-answers.md [hunk 1]
```diff
@@ -2,18 +2,67 @@
...
 
-  <iframe width="100%" height="400" src="https://www.youtube.com/embed/?listType=playlist&list=PL8K7_LTqly449uOg_uBWOPfFyL1fJRjkE&index=12" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;"></iframe>
+[//]: # ()
+[//]: # (<iframe width="100%" height="400" src="https://www.youtube.com/embed/?listType=playlist&list=PL8K7_LTqly449uOg_uBWOPfFyL1fJRjkE&index=12" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;"></iframe>)
+
+[//]: # ()
+[//]: # (---)
+# Overview
+
+You can also include relevant reference URLs in the AI assistant's responses, allowing users to access more detailed content or play related videos. In the **Existing Knowledge** page, you can find relevant knowledge and add reference URLs to enrich the AI agent's responses. This feature enhances the user experience by providing additional information and resources to users when they are looking for some specific information.
```
Comment (amy17519) [inline content/en/seachat/seachat-manual/08-add-webpage-link-in-answers.md:24]: Delete `also`. 

Change `AI assistant` to `AI agent`

**Review by amy17519 — CHANGES_REQUESTED** [review]
We need to clarify a few things on this PR:

- [ ] This feature is specifically for KB. Please move it to under KB section
- [ ] This feature is not specially for Manual Entry. All KB document types can add URL as reference. Please rewrite the section where you mentioned only Manual Entry can reference URL.  
- [ ] We actually have 3 options for advanced settings for each KB document. This PR addresses only the first one: add URL. Please plan to the wiki structure to include also the other two features, or simply add them to this PR too.
Screenshot 2024-06-05 at 21 15 07
- [ ] The is a misunderstanding about the URL reference. The URL reference does not come from the answer directly. The URL reference is triggered when the corresponding KB document is retrieved as contextual information for current answer. So when you wrote `When the user asks a question that matches the answer you have added the URL to, the AI agent will display the answer with the URL button.`, this is actually a bit misleading. Please reread your description about the URL and then make this clear.

**Comment by xuchen** [comment]
- [ ] change title to: "Adding buttons to KB responses"
- [ ] add more than 1 button
- [ ] explain that it's applicable to ALL existing documents

**Review by xuchen — APPROVED** [review]

**Review by amy17519 — APPROVED** [review]

**Pull request merged**